tests/runtests.md

```diff
@@ -40,6 +40,10 @@ Prefix a test number with a tilde (~) to still run it, but ignore the results.
 It is also possible to specify tests based on a keyword describing the test(s)
 to run, like `FTPS`. The keywords are strings used in the individual tests.
 
+Features are included as keywords with the `feat:` prefix (e.g., `feat:debug`).
+Specify a feature to run only tests with it, or exclude tests using
+`!feat:<feature>`, like `!feat:proxy`, to disable tests with that feature.
```

Comment:
s/with/requiring/ is a bit more clear I think.

Reply:
I made a slight change to the phrasing based on your suggestion. How does this sound now?

```
+ Features are included as keywords with the `feat:` prefix (e.g., `feat:debug`).
+ Specify a feature to run only tests requiring it, or exclude tests using
+ `!feat:<feature>`, like `!feat:proxy`, to disable tests which depend on that
+ feature.
```

Reply:
That's even better.